t/01-temp-files.t

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/perl
+# This program is open source, licensed under the PostgreSQL License.
+# For license terms, see the LICENSE file.
+#
+# Copyright (C) 2012-2022: Open PostgreSQL Monitoring Development Group
+
+use strict;
+use warnings;
+
+use lib 't/lib';
+use pgNode;
+use pgSession;
+use File::Path qw(rmtree);
+use Test::More tests => 39;
+
+my $node = pgNode->get_new_node('prod');
+my $proc;
+
+$node->init;
+$node->start;
+
+### Beginning of tests ###
+
+# This service can run without thresholds
+
+# Tests for PostreSQL 8.1 and before
+SKIP: {
+    skip "testing incompatibility with PostgreSQL 8.1 and before", 3
+        if $node->version >= 8.2;
```

Comment:
According to the `check_temp_files` code and inline doc, it is compatible with 8.1+.

Reply:
```suggestion
    skip "testing incompatibility with PostgreSQL 8.1 and before", 3
        if $node->version >= 8.1;
```

Reply:
Closing this suggestion as it is off by one line